### PR TITLE
LPS-117482 Update test

### PR DESCRIPTION
**Pull request by alinedoleron**
Issue: https://issues.liferay.com/browse/LPS-117482

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/chart/bar/CustomizedAxisTick.es.js [hunk 1]
```diff
@@ -17,10 +17,20 @@ import {Text} from 'recharts';
...
-			? ellipsize(payload.value, 34)
+const MAX_SIZE = 34;
+
+const minimize = (size) => (size > 5 ? MAX_SIZE / 2 : MAX_SIZE);
+
+export default ({payload, size, x, y}) => (
+	<Text
+		textAnchor="middle"
+		verticalAnchor="start"
+		width={220 - size * 17}
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/chart/bar/CustomizedAxisTick.es.js:28]: @alinedoleron, what these magic numbers `220` and `17` mean? Can we make them constants as `MAX_SIZE`?

**Comment by alinedoleron** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/chart/bar/CustomizedAxisTick.es.js (hunk 1 above)
Comment (alinedoleron) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/chart/bar/CustomizedAxisTick.es.js:28]: I'll refactor the code

**Comment by alinedoleron** [comment]
New PR: https://github.com/natocesarrego/liferay-portal/pull/690

**Pull request closed without merging**